admin/services/data-access/pupil-restart.data.service.js

```diff
@@ -159,4 +159,26 @@ pupilRestartDataService.sqlMarkRestartAsDeleted = async (pupilId, userId) => {
   return sqlService.modify(sql, params)
 }
 
+/**
+ * Find all active restarts based on pupil ids
+ * @param pupilIds
+ * @return {Promise<*>}
+ */
+pupilRestartDataService.sqlFindByPupilIds = async (pupilIds) => {
```

Comment:
This appears to be a test-developer function looking at the security (or rather the lack of the school parameter) - so do we want to separate it out into the psychometric report data service?

I started this pattern after discussion on the security fix bug.

Reply:
@jon-shipley done